scripts/assign.py

```
@@ -1,456 +1,27 @@
-                    requests.post(
-                        comments_url,
-                        headers=headers,
-                        json={"body": unassign_message},
-                    )
-                    print("Unassign comment posted.")
-                else:
-                    print(f"Issue #{issue_number} lacks 'assigned' label, skipping revocation.")
-    except Exception as e:
-        print(f"Failed to process inactive assignments: {str(e)}")
 
+def process_command(comment, assignees):
```

Ruff (0.8.2)

Missing return type annotation for public function `process_command`

Add return type annotation: `None`

(ANN201)